### ISSUE TITLE
Timezone definitions should be corrected

### ISSUE THREAD

**Issue by wisammechano** [spencermountain/timezone-soft#21]
Beside some of the missing timezone names, I see that this library incorrectly identifies the Etc timezones. For example Etc/GMT+4, it returns an offset of +4. But it should actually be -4 according to this [definition](https://github.com/eggert/tz/blob/2018d/etcetera#L36-L42) and [this](https://en.wikipedia.org/wiki/Tz_database#Area).

In node (or browser), that can actually be tested as follows:

```js
let time = new Intl.DateTimeFormat('en-GB', { dateStyle: 'full', timeStyle: 'long', timeZone: 'Etc/GMT+4' }).format(new Date());

console.log(time); // 'Tuesday, 18 April 2023 at 21:46:27 GMT-4'
```
It is also missing the Etc/GMT-14 definition. 

It would be nice to have these. I am using the library to standardize the timezones I am working with, and sometimes I need to handle specific UTC offsets and not actual geographical timezones.

**Comment by spencermountain** [comment]
hey Wisam! Thank you - I could use some help. 

First - oh my gosh, these gmt zones are inverted. Good catch.

Second - yes, let's add -14. Will do this today.

Third - Your timing is really good. I am working in earnest on a re-write of spacetime and am trying to validate iana changes and could really use some help, if you were interested.

Let me make a few changes this afternoon, and you can check them out for me.
cheers

**Comment by spencermountain** [comment]
fixed in 1.5.0 - let me know if you see any other issues
thanks
